projects/ngx-d3timeline/src/lib/axis/resources-axis/resources-axis.component.ts

```diff
@@ -11,7 +11,7 @@ import { AxisService } from '../axis.service';
       <text
         *ngFor="let tick of axis.ticks"
         [attr.transform]="tick.transform"
-        dy="-2"
+        [attr.text-anchor]="tick.textAnchor"
```

Comment:
tick doesn't have a `textAnchor` property

Reply:
Strange I inadvertently removed it, it is still set in AxisService.getTicks(..) and for some reason it doesn't cause a compile error. I will add to Tick.